courses/free-courses-pt_BR.md

```diff
@@ -113,7 +113,7 @@
 
 ### Elixir
 
-* [Curso de Elixir](https://www.youtube.com/playlist?list=PLydk1OOOmzo8VBeU334j4R4WvSByRNpXR) - ELLY ACADEMY
+* [Curso de Elixir](https://m.youtube.com/playlist?list=PLydk1OOOmzo-AtU2l102ooounamleyMB9&si=nSA4wcNbH1dgn7ZU) - ELLY ACADEMY
```

Comment:
Don't invent titles! Use the title given by the resource.
remove tracking parameter (`si`)
dont use mobil link